text/0000-non-ascii-idents.md

```diff
@@ -93,6 +109,23 @@ The confusable detection algorithm is based on [Unicode® Technical Standard #39
 
 Note: A fast way to implement this is to compute `skeleton` for each identifier once and place the result in a hashmap as a key. If one tries to insert a key that already exists check if the two identifiers differ from each other. If so report the two confusable identifiers.
 
+## Adjustments to the "bad style" lints
+
+Rust [RFC 0430] establishes naming conventions for Rust ASCII identifiers. The *rustc* compiler includes lints to promote these recommendations.
+
+The following names refer to Unicode character categories:
+
+* `Ll`: Letter, Lowercase
+* `Lu`: Letter, Uppercase
+
+These are the three different naming conventions and how their corresponding lints are specified to accommodate non-ASCII codepoints:
+
+* UpperCamelCase/`non_camel_case_types`: The first codepoint must not be in `Ll`. Underscores are not allowed except as a word separator between two codepoints from neither `Lu` or `Ll`.
```

Comment:
I think it's worth documenting that this is *already* the rule, to allow for names like `X86_64` where the underscore separates numbers, so it won't require any change to also work for underscore-separated unicameral letters.

Reply:
This whole section only describes the status-quo, it does not require changes to the compiler.